CTP/sql/bin/interactive.sh

```diff
@@ -48,6 +62,11 @@ function run()
    if [ !"$client_charset" ];then
 	client_charset=$client_charset_in_interactive
    fi
+
+   os_type=`get_os|grep Windows|wc -l`
```

Comment:
as is:
os_type=`get_os|grep Windows|wc -l`

to be:
is_cygwin_path=`uname|grep CYGWIN| wc -l`